### PR TITLE
Enable instant on for jaxws/xmlws and repeat tests for EE9 and EE10

### PR DESCRIPTION
**Pull request by anjumfatima90**
for issue #26381

### CODE REVIEW

**Comment by anjumfatima90** [comment]
#build

File: dev/io.openliberty.checkpoint_fat_xmlws/fat/src/io/openliberty/checkpoint/jaxws/fat/EJBWSBasicTest.java [hunk 1]
```diff
@@ -44,6 +48,8 @@
 @SkipIfCheckpointNotSupported
 public class EJBWSBasicTest {
 
+    private static final String SERVER_NAME = "EJBWSBasicServer";
+
     @Server("EJBWSBasicServer")
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat_xmlws/fat/src/io/openliberty/checkpoint/jaxws/fat/EJBWSBasicTest.java]: Should use new constant for `SERVER_NAME` here

Comment (anjumfatima90) [reply dev/io.openliberty.checkpoint_fat_xmlws/fat/src/io/openliberty/checkpoint/jaxws/fat/EJBWSBasicTest.java]: Done!

**Review by tjwatson — DISMISSED** [review]

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#libby

**Review by tjwatson — APPROVED** [review]

**Comment by anjumfatima90** [comment]
#libby

**Pull request merged**